src/status_im/chat/events.cljs

```diff
@@ -162,8 +162,7 @@
   (if (chats const/console-chat-id)
     {:db db}
     (cond-> {:db (-> db
-                     (assoc :new-chat sign-up/console-chat
```

Comment:
Why do we remove new-chat?

Reply:
It's just an leftover from old handler, where `:new-chat` key in db was just use to stage new-chat data-structure so it could be saved later with dedicated `side-effect!` handler into realm.
With new `fx/cofx` based handlers, there is no need for that.